### ISSUE TITLE
Bad failures cases on easy images

### ISSUE THREAD

**Issue by Xavier31** [1adrianb/face-alignment#127]
Hi Adrian. First of all thank you for this amazing work !
I am using the pre-trained model that you provide (Large) with the SFD face detector. I am testing on a in the wild dataset and i usually have good results.
However, i also obtained quite a lot of strange failures on frontal, rather easy images, like the ones attached. Does it seems like a normal behaviour to you ? I tried in 2D and 3D.
Also, results are very different from one run to another. Where does that come from ?
I am using pytorch 1.0 on the cpu on windows10
Thanks
![frame0089](https://user-images.githubusercontent.com/47775020/54211746-207ef680-44e2-11e9-86b3-8e10c4166eb6.jpg)
![frame0124](https://user-images.githubusercontent.com/47775020/54211795-34c2f380-44e2-11e9-93cc-e817f8e09214.jpg)
![frame0127](https://user-images.githubusercontent.com/47775020/54211809-3987a780-44e2-11e9-976d-4c817a297823.jpg)

These images are perfectly detected using the non deep learning face detection and landmarks from dlib (dlib own face detector and ERT from Kazemi)

**Comment by 1adrianb** [comment]
Hi @Xavier31,

Could you please try to plot also the bounding boxes? It looks to me that the face detector may have failed somehow. Can you also attach the original images that you pass to the network please? I will check to see what may have caused this since this should not happen, given how easy the images are (on a linux however, tho there should be no difference really). 
Thanks!

**Comment by Xavier31** [comment]
Adrian,
thank you for your answer.
This is indeed a problem of face detection, my build of SFD seems to be unstable. I tried 2DFan with dlib's face detection and results are ok.
I have a fair share of images (10% of my database) that have face detection problems (no detection on very easy images, strange false alarms, bounding boxes completly off). I noticed this happens only for the largest ones. When I resize these images to a smaller resolution, it works. But this is not systematic as some large images also have ok results. 
Are there any assumptions in SFD about the resolution ?

BTW, if I have more than one face bounding box, I am taking the box with the largest area.
Attached some examples of wrong bounding boxes with SFD.
I am sending some original images by email.
Thanks

![frame0089](https://user-images.githubusercontent.com/47775020/54269641-03980100-457e-11e9-8929-81a8939b7863.jpg)
![frame0124](https://user-images.githubusercontent.com/47775020/54269949-9fc20800-457e-11e9-9c7a-4d275d978499.jpg)
